src/lexer/ReservedWords.ts

```diff
@@ -98,3 +97,10 @@ export const KeyWords: {[key: string]: L} = {
     true: L.True,
     while: L.While,
 };
+
+/**
+ * These reserved words can be used as object properties (but NOT as standalone vars)
+ */
+export const ReservedWordsUsableAsProperty = new Set([
```

Comment:
This is where I would expect us to add all of the other "keyword-ish" keywords. If you add them here, they will automatically cause recoverable parse errors. So we should add all of those keywords in here at some point. I can do that now if you know the list offhand, or we can just pus it off for another day.